lib/mayaUsd/fileio/jobs/meshDataReadJob.cpp

```diff
@@ -66,7 +66,8 @@ void getComponentTags(MFnMeshData& dataCreator, const UsdGeomMesh& mesh)
 #if MAYA_API_VERSION >= 20220000
 
     std::vector<UsdMayaMeshReadUtils::ComponentTagData> componentTags;
-    UsdMayaMeshReadUtils::getComponentTags(mesh, componentTags);
+    JsValue                                             meshRoundtrippingData;
+    UsdMayaMeshReadUtils::getComponentTags(mesh, componentTags, meshRoundtrippingData);
```

Comment:
The variable is not used afterward? Or is this merely an APi change where you need to provide the parameter as empty?

Reply:
I am unsure about this class. It looks like it is extracting a mesh from a USD stage, so I don't see a need to save the roundtripping data (especially since I have no idea how to save this one).